tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -3871,6 +4124,16 @@ def _fit_extra_kwargs_dict(self):
             num_bag_sets=None,
             num_stack_levels=None,
             hyperparameter_tune_kwargs=None,
+            ds_args=dict(
+                use_holdout=True,
+                detection_time_frac=1 / 4,
+                holdout_frac=1 / 9,
+                n_folds=2,
+                n_repeats=1,
+                memory_safe_fits=True,
+                clean_up_fits=True,
+                custom_val_data=None,
+            ),
```

Comment:
If it isn't supported at fit_extra time, why is part of the fit_extra_kwargs_dict?

Reply:
Because that is called/part of `_validate_fit_extra_kwargs`. I thought this dict refers to the extra kwargs of fit and not the kwargs of fit_extra. There is no function like `_validate_fit_kwargs` AFAIK.

Reply:
Moved it to `_validate_fit_kwargs`